frontend/src/components/roadmap/Roadmap.vue

```diff
@@ -175,4 +212,11 @@ onMounted(async () => {
       </div>
     </div>
   </Form>
+
+  <FileSelectModal
+    v-model:visible="fileSelectModalVisible"
+    :documents="documents"
+    :selected-documents="selectedFiles.slice()"
```

Comment:
testing in the browser.. when adding and removing attachments the result isnt consistent. maybe this should just be 
:selected-documents="selectedFiles"